eth/filters/filter_system.go

```diff
@@ -27,25 +28,42 @@ import (
 	"github.com/ethereum/go-ethereum/event"
 )
 
+const (
+	ChainFilter = iota
+	PendingTxFilter
+	LogFilter
+	PendingLogFilter
+)
```

Comment:
Could you define a filter type for these and use that instead of a plain int?

i.e.

```go
type FilterType int

const (
 	ChainFilter FilterType = iota
 	PendingTxFilter
 	LogFilter
 	PendingLogFilter
 )
```

Also please add logs as these are publicly exposed entities.